test/e2e/sagemaker/resources/xgboost_transformjob.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: TransformJob
+metadata:
+  name: $TRANSFORM_JOB_NAME
+spec:
+  transformJobName: $TRANSFORM_JOB_NAME
+  roleARN: $SAGEMAKER_EXECUTION_ROLE_ARN
+  modelName: $MODEL_NAME
+  transformInput:
+    contentType: text/csv
+    dataSource:
+      s3DataSource:
+        s3DataType: S3Prefix
+        s3URI: s3://$SAGEMAKER_DATA_BUCKET/sagemaker/batch/input-data
+  transformOutput:
+    s3OutputPath: s3://$SAGEMAKER_DATA_BUCKET/sagemaker/batch/output
+  transformResources:
+    instanceCount: 1
+    instanceType: ml.m5.large
+  tags:
+    - key: key
+      value: value
```

Comment:
Please remove. We don't have tests for tags and these are an ugly artifact of our previous tests.

Reply:
Removed it in tests for transform job for now. These are carried over from test to test when copied, so we need to fix the others too in a different review.